src/kernel/expr.h

```diff
@@ -630,6 +630,7 @@ expr const & get_app_fn(expr const & e);
 unsigned get_app_num_args(expr const & e);
 // =======================================
 
+unsigned hash(expr const & e);
```

Comment:
```suggestion
unsigned hash(expr const & e) { ... }
```

We want this to be inlined.